build_your_own_story_book.py

```diff
@@ -0,0 +1,113 @@
+"""
+You are reading a Build Your Own Story book. It is like a normal book except that choices on some pages
+affect the story, sending you to one of two options for your next page.
+These choices are really stressing you out, so you decide that you'll either always pick the first option,
+or always pick the second option.
```

Comment:
There are lines (number of characters per line) that still exceed the PEP conventions (C0301)